### ISSUE TITLE
Swin-L path and config

### ISSUE THREAD

**Issue by MatthewCodes** [SwinTransformer/Swin-Transformer-Object-Detection#75]
Is there currently no support for training a Swin-L model? I know we need both the pth file and the python config file.
I see config files for tiny, small, and base but none for large. Can we use a config like configs/swin/cascade_mask_rcnn_swin_base_patch4_window7_mstrain_480-800_giou_4conv1f_adamw_3x_coco.py with swin_large_patch4_window7_224_22k and have it work correctly?

**Comment by impiga** [comment]
Hi. To train a cascade mask rcnn with Swin-L, we could use [configs/swin/cascade_mask_rcnn_swin_base_patch4_window7_mstrain_480-800_giou_4conv1f_adamw_3x_coco.py](https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/blob/master/configs/swin/cascade_mask_rcnn_swin_base_patch4_window7_mstrain_480-800_giou_4conv1f_adamw_3x_coco.py) and modify the backbone and neck like this.
```
backbone=dict(
    embed_dim=192,
    depths=[2, 2, 18, 2],
    num_heads=[6, 12, 24, 48],
    window_size=7,
    ape=False,
    drop_path_rate=0.3,
    patch_norm=True,
    use_checkpoint=False
),
neck=dict(in_channels=[192, 384, 768, 1536])
```

**Comment by cenchaojun** [comment]
> Thank you for your outstanding work, I only have 8 A-100 cards, can I train a batchsize each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of swin-L?

I tested it, but the segmentation was only 40.8 accurate

**Comment by alaap001** [comment]
> Hi. To train a cascade mask rcnn with Swin-L, we could use [configs/swin/cascade_mask_rcnn_swin_base_patch4_window7_mstrain_480-800_giou_4conv1f_adamw_3x_coco.py](https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/blob/master/configs/swin/cascade_mask_rcnn_swin_base_patch4_window7_mstrain_480-800_giou_4conv1f_adamw_3x_coco.py) and modify the backbone and neck like this.
> 
> ```
> backbone=dict(
>     embed_dim=192,
>     depths=[2, 2, 18, 2],
>     num_heads=[6, 12, 24, 48],
>     window_size=7,
>     ape=False,
>     drop_path_rate=0.3,
>     patch_norm=True,
>     use_checkpoint=False
> ),
> neck=dict(in_channels=[192, 384, 768, 1536])
> ```

what about detection with htc++?
